examples/gcp_defaults/.gitignore

```diff
@@ -0,0 +1 @@
+credentials.json
```

Comment:
I'd argue that this file (the `.gitgnore`) should not be in here as you have no way of knowing that the credentials file will always be called `credentials.json` - as such, the protection this file offers is contingent on the credentials file naming - to me that's fragile

Reply:
Yea i was debating that.  Thought it would be good to have an example but i deffo was thinking along your lines.